client-src/elements/feature-link.js

```diff
@@ -0,0 +1,88 @@
+import {html} from 'lit';
+// LINK_TYPES should be consistent with the server link_helpers.py
+const LINK_TYPE_CHROMIUM_BUG = 'chromium_bug';
+
+
+function enhanceChromeStatusLink(featureLink) {
+  function _formatTimestamp(timestamp) {
+    const date = new Date(timestamp * 1000);
+    const formatOptions = {
+      weekday: 'long', year: 'numeric', month: 'long', day: 'numeric',
+      hour: 'numeric', minute: 'numeric'}; // No seconds
+    return date.toLocaleString('en-US', formatOptions);
+  }
+
+  const information = featureLink.information;
+  const statusRef = information.statusRef;
+  const reporterRef = information.reporterRef;
+  const ownerRef = information.ownerRef;
+  // const ccRefs = information.ccRefs;
+  const openedTimestamp = information.openedTimestamp;
+  const closedTimestamp = information.closedTimestamp;
+  function renderTooltipContent() {
+    return html`<div class="feature-link-tooltip">
+    ${openedTimestamp && html`
+      <div>
+        <strong>Opened:</strong>
+        <span>${_formatTimestamp(openedTimestamp)}</span>
+      </div>
+    `}
+    ${closedTimestamp && html`
+      <div>
+        <strong>Closed:</strong>
+        <span>${_formatTimestamp(closedTimestamp)}</span>
+      </div>
+    `}
+    ${reporterRef && html`
+      <div>
+        <strong>Reporter:</strong>
+        <span>${reporterRef.displayName}</span>
+      </div>
+    `}
+    ${ownerRef && html`
+        <div>
+          <strong>Owner:</strong>
+          <span>${ownerRef.displayName}</span>
+        </div>
+    `}
+    </div>`;
+  }
+  return html`<div class="feature-link">
+    <sl-badge variant="${statusRef.meansOpen ? 'success' : 'neutral'}">${statusRef.status}</sl-badge>
+    <sl-tooltip style="--sl-tooltip-arrow-size: 0;">
+        <div slot="content">${renderTooltipContent()}</div>
+        <a href="${featureLink.url}" target="_blank" rel="noopener noreferrer">
+        ${information.summary}
```

Comment:
Actually, I changed my mind after testing this out myself.
Now I think that we should continue to show the URL as a URL.  The information.summary should go into the tooltip.

The reason is that our users will be entering URLs into the editing form, so I would like them to be able to see that the URL that they entered on the feature detail page as visible text. 

Also, some information.summary values might be quite confusing to first time users who don't realize that text is coming from a different system.   E.g., an issue summary line might include HTML markup or emojis or phrases like "missing information" that could be misunderstood as text that is part of our UI.

Reply:
Make sense, I will push this change at a later time.

Reply:
Yes, I think the badge can stay where it is.  Maybe the badges would look better if they were a little smaller.